test/e2e/framework/framework.go

```diff
@@ -208,6 +208,45 @@ func waitForPodsReady(client v1client.CoreV1Interface, timeout time.Duration, ex
 	}
 }
 
+func (f *Framework) WaitForServiceReady(timeout time.Duration, serviceName string) error {
```

Comment:
looks like we're duplicating these ticker functions, if you think we're going to keep adding these we should maybe extract the common ticker code